versions/1.1.0-dev.md

```diff
@@ -102,6 +102,7 @@ The metadata MAY be used by the clients if needed.
 | ---- | :----: | ---- |
 | <a name="info-title"></a>title | `string` | **REQUIRED**. A human readable description of the purpose of the overlay. |
 | <a name="info-version"></a>version | `string` | **REQUIRED**. A version identifier for indicating changes to the Overlay document. |
+| <a name="info-description"></a>description | `string` | A description of the Overlay Document. [[CommonMark]] syntax MAY be used for rich text representation. |
```

Comment:
`description` should be after `title` and before `version `

Reply:
I wasn’t aware of that, OpenAPI does it differently, and I looked at its Info Object.